src/components/UserLabel/UserLabel.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import {Envelope, Xmark} from '@gravity-ui/icons';
+
+import {Avatar} from '../Avatar';
+import type {AvatarProps} from '../Avatar';
+import {Icon} from '../Icon';
+import {block} from '../utils/cn';
+
+import i18n from './i18n';
+import type {UserLabelProps} from './types';
+import {extractTextValue, extractTextView} from './utils';
+
+import './UserLabel.scss';
+
+const COMMON_AVATAR_PROPS: Partial<AvatarProps> = {
+    size: 's',
+};
```

Comment:
This can be removed I guess?

Reply:
I think it is better to fixate avatar size